doc/user/content/release-notes.md

```diff
@@ -159,6 +159,10 @@ List new features before bug fixes.
 
 - Support PgJDBC's `getColumns()` API.
 
+- Add reserved keyword `COMPILED`.
```

Comment:
Per offline discussion: it's actually not a reserved keyword—it's an unreserved keyword—so no need for a release note!